src/simplewallet/simplewallet.cpp

```diff
@@ -2493,6 +2528,11 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
         if (dust_in_fee != 0) prompt << boost::format(tr(", of which %s is dust from change")) % print_money(dust_in_fee);
         if (dust_not_in_fee != 0)  prompt << tr(".") << ENDL << boost::format(tr("A total of %s from dust change will be sent to dust address")) 
                                                    % print_money(dust_not_in_fee);
+        if (transfer_type == TransferLocked)
+        {
+          float days = locked_blocks / 720.0f;
+          prompt << boost::format(tr(".\nThe unlock time block %llu, in approximately %s days (assuming 2 minutes per block)")) % ((unsigned long long)unlock_block) % days;
+        }
         prompt << tr(".") << ENDL << tr("Is this okay?  (Y/Yes/N/No)");
```

Comment:
but implies you can type only Y and N. Doesn't really matter though I guess. I'd ack such a change.